package.json

```diff
@@ -26,7 +26,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "android-versions": "^1.7.0",
-    "cordova-common": "^4.0.2",
+    "cordova-common": "https://github.com/liyamahendra/cordova-common",
```

Comment:
If it requires a change in cordova-common, that's fine but just writing a note here so that we don't merge the PR in while this is set to your fork.

It just means that we will have to get a PR into cordova-common and a build a release before we can merge this PR in.

Reply:
Sounds good!

Reply:
> Apart from changes in this project, I had to do a small change in cordova-common, the details of which can be seen [here](https://github.com/liyamahendra/cordova-common/blob/8f415cfa2e304692f3d4c783e8fa916b65d072b7/src/ConfigParser/ConfigParser.js#L570).

I think you can build a PR for that against cordova-common

Reply:
I think I'll revert this change, since we both are aware that a change is required in `cordova-common`.. what you do prefer?

Reply:
Why was the common PR closed?

Reply:
Sorry @erisu just reopened it.